.env.example

```
@@ -81,13 +81,11 @@ DATABASE_URL=r2dbc:postgresql://postgresql:5432/cvix
 # JDBC URL for Liquibase migrations (Dual Driver Strategy)
 # Liquibase requires traditional JDBC driver, not R2DBC
 LIQUIBASE_URL=jdbc:postgresql://postgresql:5432/cvix
-# The application may source the username/password from POSTGRESQL_* vars below.
-DATABASE_USERNAME="${POSTGRESQL_USER}"
-DATABASE_PASSWORD="${POSTGRESQL_PASSWORD}"
+# Database credentials (used by all database-dependent services)
+DATABASE_USERNAME=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_USERNAME key should go before the LIQUIBASE_URL key

(UnorderedKey)